src/all/myreadingmanga/src/eu/kanade/tachiyomi/extension/all/myreadingmanga/MyReadingManga.kt

```diff
@@ -29,7 +28,6 @@ open class MyReadingManga(override val lang: String, private val siteLang: Strin
     // Basic Info
     override val name = "MyReadingManga"
     final override val baseUrl = "https://myreadingmanga.info"
-    override val client: OkHttpClient = network.cloudflareClient
```

Comment:
We are still using ext-lib 1.4 so probably better to leave cloudflare client for now

Reply:
Done